SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/EventSearchAssetVoltageDisturbances.tsx

```diff
@@ -58,7 +58,7 @@ const EventSearchAssetVoltageDisturbances: React.FC<SEBrowser.IWidget<any>> = (p
     }
 
     return (
-        <div className="card">
+        <div className="card" style={{ maxHeight: props.maxHeight ?? 500, overflowY: 'auto' }}>
```

Comment:
This should be in the `tbodyStyle` property of the table, not the card. This makes the entire card scroll, including card and table headers.